routes/web.php

```diff
@@ -84,13 +84,21 @@
         ->name('device')->where('vars', '.*');
 
     // Maps
-    Route::prefix('maps')->group(function () {
+    Route::get('fullscreenmap', 'Maps\FullscreenMapController@fullscreenMap');
+    Route::get('availability-map', 'Maps\AvailabilityMapController@availabilityMap');
+    Route::get('map/{vars?}', 'Maps\NetMapController@netMap');
+    Route::prefix('maps')->namespace('Maps')->group(function () {
```

Comment:
Instead of namespace Maps here, you could just use the newer array syntax.  I don't feel that strongly about it though.